rules-reviewed/jboss-eap/glassfish/xml-glassfish.windup.xml

```diff
@@ -24,13 +24,15 @@
                 </xmlfile>
             </when>
             <perform>
-                <iteration>
-                    <classification title="Glassfish Web Descriptor" effort="3" severity="mandatory">
-			<description> Glassfish specific web descriptor file that is by convention named glassfish-web.xml. It is used to configure web application properties likes cache, session configuration, JNDI name binding etc.</description>
-		    <link href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.2/html/Security_Guide/jboss-webxml_Configuration_Reference.html" title="JBoss EAP web descriptor"/>
-                    <tag>configuration</tag>
-		    </classification>
-                </iteration>
+               <iteration>
+                  <classification title="Glassfish Web Descriptor" effort="3" severity="mandatory">
+                     <description> Glassfish specific web descriptor file that is by convention named glassfish-web.xml. It is used to configure web application properties likes cache, session configuration, JNDI name binding etc.</description>
+                     <link
+                        href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.2/html/Security_Guide/jboss-webxml_Configuration_Reference.html"
+                        title="JBoss EAP web descriptor" />
+                     <tag>configuration</tag>
```

Comment:
not sure as metadata has got glassfish tag, I thought it is added as global tag, isn't it?